src/DynamicFormControlGroup.vue

```diff
@@ -1,30 +1,34 @@
 <template>
-    <b-row class="my-2">
-        <label v-if="controlGroup.label" class="col-form-label col-md-5">{{controlGroup.label}}
+    <conditional-label
+        :readonly="readonly"
+        :control-group="controlGroup"
+    >
+        <div class="col-form-label col-md-5">
```

Comment:
Would be good if this wasn't a `<div>`... in HTML it's not valid for `<label>` or `<span>` elements to contain divs (I checked by passing the new rendered HTML through the W3C's validator). span should be ok instead.